libraries/c_sdk/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -896,9 +896,9 @@ static int32_t transportSend( const NetworkContext_t * pNetworkContext,
     /* Sending the bytes on the network using Network Interface. */
     bytesSend = pNetworkContext->pNetworkInterface->send( pNetworkContext->pNetworkConnection, ( const uint8_t * ) pMessage, bytesToSend );
 
-    if( bytesSend < 0 )
+    if( bytesSend <= 0 )
```

Comment:
Is there a unit test to cover this case?  If so, does the test need to be updated?

Reply:
Thanks. No there is no unit test case covering this case for the MQTT Shim.